test/cli/test_login.rb

```diff
@@ -0,0 +1,237 @@
+require_relative "test_helper"
+require "aruba"
+
+class CLITest < Minitest::Test
+  include Aruba::Api
+  include Helpers
+
+  def setup
+    setup_aruba
+  end
+
+  def test_login
+    label = "test-login"
+    username = "user"
+    password = "pass"
+    output = <<~OUTPUT
+      Label: #{label}
+      \tUsername: #{username}
+      \tPassword: #{password}
+    OUTPUT
+
+    # create login:
+    run_command "kbsecret new login #{label}"
```

Comment:
Let's use the block form of `run_command`, since it makes it easier to transpose test cases around and lets us distinguish between stopping a command and waiting for it to finish:

```ruby
run_command "kbsecret new login #{label}" do |cmd|
  cmd.stdin.puts username, password
  cmd.stdin.close
  cmd.wait
end
```

Reply:
Oh, and as a general note: you don't need to interpolate variables that are already strings.

These two lines are equivalent:

```ruby
type "#{foo}"
type foo
```